web/template/certificate_provider_check_your_name.gohtml

```diff
@@ -0,0 +1,54 @@
+{{ template "page" . }}
+
```

Comment:
need to add the `{{ define "serviceName" }}{{ tr .App "beACertificateProvider" }}{{ end }}` block in

Reply:
I'd totally missed we were even doing this - added in now.